run.sh

```diff
@@ -1,8 +1,30 @@
 #!/bin/sh
 
+#add environmental variable of .env to the bash script
+. ./.env
+
+#Define absolute path for the host and the serv from the relative common path
+HOST_LOG_FILE=$(realpath -P ${LOG_FILE})
```

Comment:
You should check first that `LOG_FILE` isn't empty!
You can easily do that in bash with 
```sh
[ -z "${LOG_FILE}"] && {echo "a good error msg"; exit 1;}
```
the `-z` means empty if I'm not mistaken!

Reply:
done in 08eb12e